Express-API/roles.js

```diff
@@ -57,5 +78,30 @@ class RolesHandler {
             }
         })
     }
+
+    viewEditRole(params, db){
+        return new Promise( ( resolve, reject ) => {
+            try {
+                db.query(viewEditRole,[params.roleID]).then(rows => {
+                    resolve(rows)
+                })
+            } catch (err){
+                reject({"error":"Error getting role information"})
+            }
+        })
+    }
+
+    roleBandCapabilityExists(params, db){
+        return new Promise( ( resolve, reject ) => {
+            try {
+                db.query(RoleBandCapabilityExists,[params.capabilityId, params.bandId]).then(rows => {
+                    resolve(rows)
+                })
+            } catch (err){
+                reject({"error":"Error getting role information"})
+            }
+        })
+    }
+
 }
-module.exports = RolesHandler
\ No newline at end of file
+module.exports = RolesHandler
```

Comment:
Something worth testing here, but I think you can change this to `export default new RolesHandler()`.

`module.exports` is used in older versions of JS, and because yous are using JS version ES2018/ES9 (specified here, https://github.com/sapphireduffy/KainosCareerLattice/blob/master/Angular/tsconfig.json#L18), you can use this newer syntax.

See https://stackoverflow.com/a/46677972.